birdhouse/pavics-compose.sh

```diff
@@ -129,10 +129,10 @@ if [ x"$1" = x"up" ]; then
   echo ${COMPOSE_CONF_LIST} | tr ' ' '\n' | grep -v '^-f'
 
   # the PROXY_SECURE_PORT is a little trick to make the compose file invalid without the usage of this wrapper script
-  PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker-compose --project-directory "${BUILD_DIR}" ${COMPOSE_CONF_LIST} config -o "${COMPOSE_FILE}"
+  PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker compose --project-directory "${BUILD_DIR}" ${COMPOSE_CONF_LIST} config -o "${COMPOSE_FILE}"
```

Comment:
To be on the safe side, I would suggest leaving a variable that resolve the command with a fallback to the old name.
Something like:
```shell
DOCKER_COMPOSE=$(docker compose version >/dev/null && echo "docker compose" || echo docker-compose)
```

Reply:
I've removed this change for now. I think that we should do it eventually but there is a chance that it will have unintended side effects that we don't detect here. And I'd rather not deal with that in this PR